{"name": "Проверка получения ошибки аутентификации курьера при вводе невалидных данных", "status": "passed", "description": "В тест по очереди передаются наборы данных с несуществующим логином или неверным паролем. Проверяются код и тело ответа.", "parameters": [{"name": "nonexistent_credentials", "value": "{'login': 'Ground.898', 'password': 'ZSUGiOMo^4'}"}], "start": 1723036926014, "stop": 1723036926294, "uuid": "8d4eb354-1d5a-495f-b357-2ffaafab2027", "historyId": "64f0fbfa4e14683d719f41ae656ea0d4", "testCaseId": "199be2f2af7455b14ed4795a4ee970da", "fullName": "tests.test_courier_login.TestCourierLogin#test_courier_login_nonexistent_data_not_found", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "DESKTOP-82VEM3U"}, {"name": "thread", "value": "12784-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}